proto/one/enterprise/data/report_definition_json_v2.proto

```diff
@@ -15,9 +16,7 @@ message ReportDefinitionJsonV2 {
 
     EnumReportDefinitionCategory category = 1;
 
-    repeated EnterpriseDataTableQueryParameter queryParameters = 2;
+    repeated EnterpriseDataTableQueryParameter additionalContent = 2;
```

Comment:
renumbering is a breaking change, is anyone using this yet that could be impacted?

Reply:
Yeah, I ran this by @JonathanRobertson-AQI. The change shouldn't affect anyone since it's not used yet. The original was created earlier this sprint.